Time and Time Again/src/TimeAndTimeAgain.java

```diff
@@ -0,0 +1,40 @@
+import java.io.File;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+import java.io.File;
+import java.io.IOException;
+
+public class TimeAndTimeAgain
+	{
+		public static void main(String[] args) throws IOException
+			{
+			Scanner input = new Scanner(new File("TimeAndTimeAgainTextfile.txt"));
+			int numberOfCases = input.nextInt();
```

Comment:
So after fixing the regex sting patterns below, there's an issue where when I enter the first number (say I enter 2), it prints out an empty "0:0:0:" immediately instead of waiting for further input. I haven't had a chance to look into this more, but replacing` input.nextInt()` with `Integer.parseInt(input.nextLine())` prevents it from printing that extra timestamp. I'll see if I can get a better explanation for this other than "it's magic"

Reply:
Oh interesting, the copy I pulled down is different, it's not reading in a file but from the standard input. If you're having an issue with reading in a file as well, I would maybe try changing the input line here and see if that fixes it as well.